pkg/config/api/v1beta1/types.go

```diff
@@ -5,12 +5,11 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 )
 
-// Config contains a set of Kubernetes resources to be applied.
-// TODO: Unify with Kubernetes Config
-//       https://github.com/GoogleCloudPlatform/kubernetes/pull/1007
+// Config contains a list of Kubernetes resources to be applied.
+// DEPRECATED: All clients should use Kubernetes api.List instead of Config.
 type Config struct {
-	kapi.TypeMeta   `json:",inline"`
-	kapi.ObjectMeta `json:"metadata,omitempty"`
+	kapi.TypeMeta `json:",inline"`
+	kapi.ListMeta `json:"metadata,omitempty"`
```

Comment:
it was not :-) I changed it

On Wed, Jan 28, 2015 at 3:41 PM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/config/api/v1beta1/types.go
> <https://github.com/openshift/origin/pull/698#discussion_r23690563>:
>
> >  type Config struct {
> > -	kapi.TypeMeta   `json:",inline"`
> > -	kapi.ObjectMeta `json:"metadata,omitempty"`
> > +	kapi.TypeMeta `json:",inline"`
> > +	kapi.ListMeta `json:"metadata,omitempty"`
>
> Yes that's good
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/698/files#r23690563>.
>



-- 
-----------------------------------------------
Michal Fojtik <mi@mifo.sk <mfojtik@mifo.sk>>
Red Hat OpenShift, Engineering